eZ/Publish/API/Repository/Tests/LimitationServiceTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Tests;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use eZ\Publish\Core\Base\Exceptions\BadStateException;
+use eZ\Publish\Core\Base\Exceptions\NotFound\LimitationNotFoundException;
+use eZ\Publish\Core\FieldType\ValidationError;
```

Comment:
Integration tests _should_ not know about implantation, so if possible only type hint with interfaces from 1. API 2. SPI